### ISSUE TITLE
Shorten "toot" to "t"

### ISSUE THREAD

**Issue by booyaa** [magicalraccoon/tootstream#60]
I'm lazy, don't want to type out toot every time.

**Comment by brrzap** [comment]
this is on my wishlist.  generally providing short aliases for the REPL commands and/or tab completion as #59 requests.  i think Click can provide these but i haven't quite figured out how.

**Comment by brrzap** [comment]
i have an experimental branch with click_shell applied on top of the recent rollup if you want to play with it.  command completion and aliases (no argument completion but it may be possible this way).  you'll need to rerun the setup in your virtualenv after cloning the branch or `pip install click_shell==1.0`.  https://github.com/brrzap/tootstream/tree/experimental-clickshell

**Comment by booyaa** [comment]
just given your experimental branch a go, it's awesome. are you happy for me to log feedback via issues on your own fork?

**Comment by brrzap** [comment]
yes, that's fine.  i don't know how much of that stuff might be of interest to the mainline, so let's not clutter up the issues here.
